lynx/patterns/link-in-bio-cloud.php

```diff
@@ -1,18 +1,15 @@
 <?php
 /**
- * Title: A Link in Bio pattern with link cloud
+ * Title: Link in Bio with link cloud
  * Slug: lynx/link-in-bio-cloud
  * Inserter: yes
  * Categories: featured, link-in-bio
  */
-
 ?>
 
 <!-- wp:cover {"overlayColor":"white","minHeight":100,"minHeightUnit":"vh","contentPosition":"center center","isDark":false,"align":"full"} -->
 <div class="wp-block-cover alignfull is-light" style="min-height:100vh"><span aria-hidden="true" class="wp-block-cover__background has-white-background-color has-background-dim-100 has-background-dim"></span><div class="wp-block-cover__inner-container"><!-- wp:group {"style":{"spacing":{"blockGap":"0px"}},"layout":{"inherit":false,"contentSize":"640px"}} -->
-<div class="wp-block-group"><!-- wp:paragraph {"align":"left","placeholder":"Write title…","style":{"color":{"text":"#00000099"}},"fontSize":"small"} -->
-<p class="has-text-align-left has-text-color has-small-font-size" style="color:#00000099"><?php echo  esc_html__( 'I’m Matt, an artist living in New York City.', 'lynx' ); ?></p>
-<!-- /wp:paragraph -->
+<div class="wp-block-group"><!-- wp:site-tagline {"style":{"color":{"text":"#00000099"}},"fontSize":"1rem"} /-->
```

Comment:
Is this hex code correct: `#00000099`?

Reply:
Ahh OK, this must be an upstream bug then (hex codes should only be 3 or 6 characters long). I'll search for / open a GB issue.

It's probably working as I think it'll default to black/#000000 (the first 6 characters).

Reply:
> It's probably working as I think the browser will default to black/#000000 (the first 6 characters).

When I insert this pattern, the editor picks up the opacity setting correctly - doesn't switch to #000000.

Reply:
I guess 8 digit HEX codes are a thing now: https://caniuse.com/?search=hex

Reply:
Ooh I see, it's a way to set the alpha! I usually see it in this format: `rgba(0, 255, 0, 0.5)`. I'm less confused now, thanks! 😁